.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.experimental }}
+    name: "Build and Test on ${{ matrix.python-version }}, experimental: ${{ matrix.experimental }}"
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.9", "3.10", "3.11"]
+        experimental: [false]
+        # include:
+        #   - python-version: "3.11"
+        #     experimental: true
```

Comment:
reminder

Reply:
oh, i've been leaving this in order to start testing on python 3.12 when it comes out as experimental